DOCS-11370 credential formula (#233)
mulesoft/docs-api-community-manager: modules/ROOT/pages/manual-setup.adoc

. Set *Token Endpoint Url* to `+https://anypoint.mulesoft.com/accounts/api/v2/oauth2/token+`.
. Set *Issuer* to the client ID of the connected app you saved previously.

> . Set *Per User Subject* to the following formula.

Reviewer: Where is "the following formula"?

Author: These are the formulas on lines 94 and 99.

> . Set *Per User Subject* to the following formula.
> +
> The guided setup uses a slightly different formula in the `IDP_ISSUER` field, but either formula works.
> +

Reviewer: ```suggestion
. Set *Per User Subject* to one of the following formulas:
+

```

Author: Thank you! Fixed.
